charts/metallb/templates/webhooks.yaml

```diff
@@ -12,7 +12,7 @@ webhooks:
       name: metallb-webhook-service
       namespace: {{ .Release.Namespace }}
       path: /validate-metallb-io-v1beta1-addresspool
-  failurePolicy: Fail
+  failurePolicy: {{ default "Fail" .Values.crds.validationFailurePolicy }}
```

Comment:
I think there's no need to specify the default here, as long as we add it in values.yaml
same for the following.
cc @gclawes

Reply:
fair point, it's unnecessary duplication of the default value, updated